packages/common/test-configs/jest-config-base.ts

```diff
@@ -13,6 +13,7 @@ const config: InitialOptionsTsJest = {
     '\\.(css|scss)$': 'identity-obj-proxy',
   },
 
+  passWithNoTests: true,
```

Comment:
FWIW I originally added this to our package.json as the publish step usually failed for lib-core etc since there were no tests. We probably eventually should change this to only be packages we expect there to be no tests for, versus blanket allowing it across the board.

Reply:
Well, if a package has no tests, we can either fail the build, or let it pass.

I think letting it pass is more sensible in terms of defaults. The default imposed by Jest is more along the lines of "you have a project with test infrastructure set up, not having any tests doesn't make much sense, therefore it's treated as an error".

Right now, `lib-core` and `lib-webpack` are missing tests but we will add them later on. As for the upcoming `lib-extensions` whose scope is very narrow, tests may not even be necessary for that package, unless we add some logic or we want to get really fancy and start "unit testing" the TS types 😃 also consider the fact that we use the same test infrastructure across multiple packages within a monorepo.

👍 on changing it later on, only for packages where we don't expect any tests.